src/cmd/burn.rs

```diff
@@ -100,7 +108,8 @@ fn print_txn(
                 "fee": txn.fee,
                 "nonce": txn.nonce,
                 "hash": status_json(status),
-                "txn": envelope.to_b64()?
+                "txn": envelope.to_b64()?,
+                "Status": status_endpoint
```

Comment:
```suggestion
                "status": status_endpoint
```

these should be lower-case to be consistent with all the other json fields

Reply:
Ok, I only changed the JSON status field to "status" as opposed to "Status".